jenkins/flow/pr/edx-platform-bok-choy-pr.groovy

```diff
@@ -6,12 +6,19 @@ def sha1 = build.environment.get("ghprbActualCommit")
 def subsetJob = build.environment.get("SUBSET_JOB") ?: "edx-platform-test-subset"
 def repoName = build.environment.get("REPO_NAME") ?: "edx-platform"
 def workerLabel = build.environment.get("WORKER_LABEL") ?: "jenkins-worker"
+def djangoVersion = build.envrironment.get("DJANGO_VERSION") ?: ""
+
+// Any environment variables that you want to inject into the environment of
+// child jobs of this build flow should be added here (comma-separated,
+// in the format VARIABLE=VALUE)
+// following v
+def envVarString = "DJANGO_VERSION=${djangoVersion}"
 
 guard{
     parallel(
       (1..11).collect { index ->
         return {
-          bokchoybuild = build(subsetJob, sha1: sha1, SHARD: index, TEST_SUITE: "bok-choy", PARENT_BUILD: "master #" + build.number, WORKER_LABEL: workerLabel)
+          bokchoybuild = build(subsetJob, sha1: sha1, SHARD: index, TEST_SUITE: "bok-choy", PARENT_BUILD: "master #" + build.number, WORKER_LABEL: workerLabel, ENV_VARS: envVarString)
```

Comment:
Nit: I don't know about groovy formatting - but is it time now to move the `build()` calls in this PR to a multi-line format?

Reply:
Good call. We currently don't have any linters set up on this repo (but plan on it someday).